### DISCUSSION TITLE
Having the hardest time with pf1e

### DISCUSSION THREAD

**Discussion by Belgaron** [mclemente/fvtt-condition-lab-triggler#53] [Q&A]
Hello, trying to get this to work with pf1e. I spent like 10+ hours playing with it a few months ago and got it working. Updated to the latest pf1e setting and it seems like all the custom trigger paths I created are wiped out. Nothing I can think of is working.

This is what I'm trying so far.
![image](https://github.com/user-attachments/assets/d054b6bb-d563-4285-bb49-e47a9ab9e6eb)

I also have a data module to help me find paths, but i swear it never works with anything
![image](https://github.com/user-attachments/assets/ecbb0c1b-6d11-46c8-99e4-e85e99159536)

Need help.

**Comment by mclemente** [comment]
Since you've lost the previous triggers, have you set them again on the conditions to be applied/removed?
